### DISCUSSION TITLE
Emoji Support

### DISCUSSION THREAD

**Discussion by Tofixrs** [emilk/egui#1934] [Q&A]
So i copy pasted an emoji into an text edit and  this shows up. (The weird box)
![image](https://user-images.githubusercontent.com/73693639/185597128-013515a2-ed2c-4a41-801a-728bc402a778.png)
Is there a way to make it a one character without using a custom font?

**Comment by Paul-Dempsey** [comment]
The "weird" character is a standard "missing glyph" character, so you apparently don't have a font in the font list that contains the particular emoji. See https://docs.rs/egui/latest/egui/struct.FontDefinitions.html for more info on providing a custom list.

So, basically, no -- you'll need to provide an emoji font to show emoji characters. Same for any other character you might want to display: you need a font that has the character.
